test/ufixed.jl

```diff
@@ -122,7 +134,7 @@ x = 0xaauf8
 iob = IOBuffer()
 show(iob, x)
 str = takebuf_string(iob)
-@test startswith(str, "UFixed8(")
+@test startswith(str, "FixedPointNumbers.UFixed{UInt8,8}(")
```

Comment:
I say truncate it. As Jeff says, printing of *types* should include all the gory details. Printing of *values* can take shortcuts.